judge/views/user.py

```diff
@@ -507,3 +513,121 @@ def post(self, request, *args, **kwargs):
             return HttpResponse(_('You have sent too many password reset requests. Please try again later.'),
                                 content_type='text/plain', status=429)
         return super().post(request, *args, **kwargs)
+
+
+class EmailChangeRequestView(LoginRequiredMixin, TitleMixin, FormView):
+    title = _('Change your email')
+    template_name = 'registration/email_change.html'
+    form_class = EmailChangeForm
+
+    activate_html_email_template_name = 'registration/email_change_activate_email.html'
+    activate_email_template_name = 'registration/email_change_activate_email.txt'
+    activate_subject_template_name = 'registration/email_change_activate_subject.txt'
+    notify_html_email_template_name = 'registration/email_change_notify_email.html'
+    notify_email_template_name = 'registration/email_change_notify_email.txt'
+    notify_subject_template_name = 'registration/email_change_notify_subject.txt'
+
+    def send_mail(
```

Comment:
Should this method be here, or should we refactor it out into some util file?

Reply:
Good point, I've refactored it.